### PR TITLE
Add java modules to list functions. Fixes #3425

### LINKED ISSUE #3425: [BUG] util:registered-functions throws XPathException with certain modules
**Describe the bug**

For 3 eXist extension modules, `util:registered-functions` throws an XPathException error.

Example error:

> No module found matching namespace URI: `http://exist-db.org/xquery/memsort`

Affected modules:

- `http://exist-db.org/xquery/console `
- `http://exist-db.org/xquery/memsort `
- `http://expath.org/ns/crypto `

**Expected behavior**

This function should not raise an error when provided with a valid extension module namespace URI.

**To Reproduce**

The following query shows which modules produce the error.

```xquery
xquery version "3.1";

for $namespace-uri in util:registered-modules()
return
    try { 
        let $functions := util:registered-functions($namespace-uri) 
        return
            ()
    } catch * { 
        <error>{$err:code} raised when getting registered functions for module {$namespace-uri}</error>
    }
```

**Context (please always complete the following information):**
 - OS: macOS 10.13.6
 - eXist-db version: 5.2.0, 5.3.0-SNAPSHOT
 - Java Version: 1.8.0_242 (Zulu 8.44.0.11-CA-macosx) (build 1.8.0_242-b20)

**Additional context**
- How is eXist-db installed? DMG, source
- Any custom changes in e.g. `conf.xml`? no

### PR DESCRIPTION
**Pull request by dizzzz**
Fixes #3425

(adam: sorry, there is a bit of mark-up changed too; I tried to rework original PR and then all fell down)

### CODE REVIEW

**Comment by adamretter** [comment]
@dizzzz I am a bit confused!

I thought that PR https://github.com/eXist-db/exist/pull/3428 fixed issue #3425 ?

**Comment by dizzzz** [comment]
adam no that was #3424 ; the description of the issues are quite similar but different.

**Comment by adamretter** [comment]
@dizzzz Okay thanks for the clarification. Can you tell me what you changed - unfortunately there is only 1 commit in this PR and there is so much reformatting that I can't see what happened.

File: exist-core/src/main/java/org/exist/xquery/functions/util/BuiltinFunctions.java [hunk 1]
```diff
@@ -3,130 +3,145 @@
...
+		if (getArgumentCount() == 1) {
 			final String uri = args[0].getStringValue();
-			final Module module = context.getModule(uri);
-			if(module == null)
-				{throw new XPathException(this, "No module found matching namespace URI: " + uri);}
+
+			// Get 'internal' modules
+			Module module = context.getModule(uri);
+
+			// If not found, try to load Java module
```
Comment (dizzzz) [inline exist-core/src/main/java/org/exist/xquery/functions/util/BuiltinFunctions.java:100]: Begin of change

**Comment by dizzzz** [comment]
@adamretter yes sorry, in an attempt to rework the PR, I made a mess :-(

The only actual changes is the `if` statement of line 100-103:

image


for a better diff.... set ignore whitespace off: https://github.com/eXist-db/exist/pull/3430/files?diff=split&w=1

**Comment by adamretter** [comment]
@dizzzz thanks that is much clearer now :-)

**Pull request merged**